api/src/utils/user-utils.js

```diff
@@ -0,0 +1,23 @@
+const removeFields = (obj, fields) => {
+  const objCopy = { ...obj };
+
+  fields.forEach((field) => {
+    delete objCopy[field];
+  });
+
+  return objCopy;
+};
+
+const filterSensitiveInfo = (user, additionalFields = []) => {
+  const omitFields = ['_id', 'oauthID', '__v', ...additionalFields];
```

Comment:
Could you put a TODO here to add 'notes' to the omit fields once we add them to the DB. If different people are working on this later, I could see that being overlooked

Reply:
Yes, good point!